shoop/admin/static_src/base/js/content-blocks.js

```diff
@@ -21,4 +21,9 @@ $(function() {
             $(this).find(".block-title").addClass("mobile-error-indicator");
         }
     });
+    // Activate first sidebar-list-item with errors
+    var $firstSidebarListItemWithErrors = $("a.sidebar-list-item.errors").first();
+    if ($firstSidebarListItemWithErrors.length > 0) {
```

Comment:
I don't think you actually even need to check for the length -- [the trigger would act on a collection of 0 element, i.e. not much at all](https://github.com/jquery/jquery/blob/master/src/event.js#L912-L916).